api.rst

```diff
@@ -1086,7 +1086,7 @@ Since it contains the ``films_id`` foreign key, it is possible to embed ``box_of
 
 .. note::
   * Partitioned tables can reference other tables since PostgreSQL 11 but can only be referenced from any other table since PostgreSQL 12.
-  * Partitions are not included in the :ref:`schema_cache` since they should not be exposed to such features. This also avoids embedding ambiguity for the parent partitioned table.
+  * Embedding on partitions is not allowed because it leads to ambiguity errors between them and their parent partitioned table (see :ref:`embed_disamb`). :ref:`custom_queries` can be used if this is needed.
```

Comment:
```suggestion
  * Embedding on partitions is not allowed because it leads to ambiguity errors (see :ref:`embed_disamb`) between them and their parent partitioned table. :ref:`custom_queries` can be used if this is needed.
```